stdlib/cmake/modules/SwiftSource.cmake

```
@@ -889,6 +889,10 @@ function(_compile_swift_files
         COMMENT "Generating ${module_file}")
 
     if(SWIFTFILE_STATIC)
+      set(command_copy_interface_file)
+      if("${interface_file}")
```

I think that this can be rewritten as:
```suggestion
      if(interface_file)
```